### DISCUSSION TITLE
Add New Idea

### DISCUSSION THREAD

**Discussion by MOHAMED-EHAB-DEV** [reactplay/react-play#174] [Ideas]
Awesome React Portfolio with Animations like this: https://amazing-portfolio-react-sanity.vercel.app/

**Comment by atapas** [comment]
You can create an issue from this discussion and make the code changes for this idea to appear on the ideas page. Let me know if you need any help.

**Reply by atapas** [reply]
@Programming-School-Pro-Coding Please let me know here once done. I shall mark this discussion as closed.

**Reply by MOHAMED-EHAB-DEV** [reply]
Ok

**Reply by MOHAMED-EHAB-DEV** [reply]
I will make the changes soon....

**Reply by MOHAMED-EHAB-DEV** [reply]
> @Programming-School-Pro-Coding Please let me know here once done. I shall mark this discussion as closed.

Mark this discussion as closed

**Comment by MOHAMED-EHAB-DEV** [comment]
Hey @atapas 

I add the idea in commits

I create a new Pull Request

Please review
